package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "css4-training",
+  "version": "1.0.0",
+  "description": "Project for training on css4 grid and flexbox",
```

Comment:
Proposal:

"Training Material for CSS4 Lemoncode course by Jaime Salas and Carmen Ansio